chia/full_node/full_node_api.py

```diff
@@ -67,7 +67,6 @@
 from chia.types.unfinished_block import UnfinishedBlock
 from chia.util.batches import to_batches
 from chia.util.db_wrapper import SQLITE_MAX_VARIABLE_NUMBER
-from chia.util.full_block_utils import header_block_from_block
```

Comment:
I imagine the actual bug is in this function, right?
*if* we don't use it anymore, it should be removed. However, I have a feeling this optimization *is* still relevant and that this bug needs to be fixed instead

Reply:
There is no bug with this function itself. It accepts removals and additions from you, so if you deliver that wrong, you get an incorrect header block.